### DISCUSSION TITLE
GetListing endpoint: In swagger definition Shipping_profile became non-nullable but endpoint still returns null

### DISCUSSION THREAD

**Discussion by MaciejDerek** [etsy/open-api#1187] [Q&A]
Hello,

I face the issue when generating c# model from openapi swagger. It seems like in the endpoint getListing the property shipping_profile became non-nullable, however still I receive null when I call this endpoint for my listing. Which then results in exception in json deserialization. Please advise.

**Comment by showtroylove** [comment]
The output of that tool is just a starting point. The code is often boilerplate in need of major refactoring depending on the generated language. In your case, specifically C#, the tool's options could cause it to emit nullable types. Check the nullability settings of your project to ensure you haven't inadvertently changed them. This issue is not related to Etsy's API or the Swagger definition but rather to your toolset. Typically, Etsy's API returns an empty Results object { long: Count, List<T>: Results } when no data is found.
